doc/Pacemaker_Explained/en-US/Ch-Advanced-Options.txt

```diff
@@ -627,6 +627,20 @@ The +ipmiservicelogd+ daemon sets node health attributes based on IPMI
 values (the +ocf:pacemaker:SystemHealth+ resource agent can be used to manage
 the daemon as a cluster resource).
 
+In order to take advantage of this feature - firstly add the resource to your cluster, preferably as a cloned resource to constatnly measure health on all nodes:
+
+----
+# pcs resource create resHealthCPU ocf:pacemaker:HealthCPU yellow_limit=50 red_limit=15 op monitor timeout="5" interval="5" clone
+----
+
+This way attrd_updater will set proper status for each node running this resource. Any attribute matching "#health-[a-zA-z]+" will force cluster to migrate all resources from unhealthy node and place it on other nodes according to all constraints defined in your cluster.
+
+When the node is no longer faulty you can force the cluster to restart the cloned resource on faulty node and make it available to take resources, in this case since we are using HealthCPU provider:
+
+----
+# attrd_updater -n "#health-cpu" -U "green" --node="<nodename>" -d "60s"
+----
```

Comment:
Just FYI, I remember a conversation where it was decided it would be a good idea to add support for a meta-attribute indicating that the resource was immune to health checks, so that we could allow health monitors to keep running on a node even after it turned red. That would allow it to recover automatically. Unfortunately there's not enough developer time to add all the would-be-nice features ...

Reply:
I suppose we can create an issue for that and prepare separate pull request for it (I am not familiar with pacemaker source code enough but maybe I will take a look into this in some time) ? It is certainly a good idea to auto-mark healthy nodes as "green" (personally I struggled for quite a while to find out how to add the node back into cluster pool).